tests/src/rules/no-named-default.js

```diff
@@ -0,0 +1,31 @@
+import { test, SYNTAX_CASES } from '../utils'
+import { RuleTester } from 'eslint'
+
+const ruleTester = new RuleTester()
+    , rule = require('rules/no-named-default')
+
+ruleTester.run('no-named-default', rule, {
+  valid: [
+    test({code: 'import bar from "./bar";'}),
+    test({code: 'import bar, { foo } from "./bar";'}),
+
+    // es7
+    test({ code: 'import bar from "./bar";', parser: 'babel-eslint' }),
+    test({ code: 'import bar, { foo } from "./bar";', parser: 'babel-eslint' }),
```

Comment:
What are the differences with the previous non-es7 test cases? Are you just testing the parsers?